maps/unused/oshan_xmas_2018.dmm

```diff
@@ -10780,7 +10780,7 @@
 	pixel_y = 5
 	},
 /obj/item/device/radio/intercom/science{
-
+	
```

Comment:
All of these blank-line changes seem unneeded?

Reply:
I can not speak to operations that were performed on unmaintained maps by automated mechanisms.... 🤖 
Owned by githook. 💀

Reply:
Do you want me to remove these changes?